### PR TITLE
Create V10 version of Azure Blob Storage documentation

### PR DESCRIPTION
**Pull request by Ambertvu**
As section about 'UseAzureBlobMediaFileSystem()' is no longer needed in v10.

### CODE REVIEW

**Comment by eshanrnh** [comment]
Thanks, @Ambertvu for the PR 💪

We will review this soon 😃

File: Extending/FileSystemProviders/Azure-Blob-Storage/index-v9.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+---
+versionFrom: 9.0.0
+versionTo: 9.x
```
Comment (sofietoft) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index-v9.md:3]:
```suggestion
versionTo: 9.0.0
```

**Review by sofietoft — COMMENTED** [review]
Thanks again for the PR @Ambertvu ! 💪 

Just a single suggestion to the versioning.
I'll make sure we have this properly reviewed as soon as possible.

**Review by bergmania — APPROVED** [review]

**Comment by sofietoft** [comment]
Hi all 👋 
Apologies for the long wait on this one.

I will make that it gets merged right away.
Thanks again for helping out 💪

**Pull request merged**